src/Simplex/Chat/Migrations/M20231008_member_settings.hs

```diff
@@ -0,0 +1,18 @@
+{-# LANGUAGE QuasiQuotes #-}
+
+module Simplex.Chat.Migrations.M20231008_member_settings where
+
+import Database.SQLite.Simple (Query)
+import Database.SQLite.Simple.QQ (sql)
+
+m20231008_member_settings :: Query
+m20231008_member_settings =
+  [sql|
+ALTER TABLE group_members ADD COLUMN show_messages INTEGER;
```

Comment:
it's enable_ntfs in contacts and groups

Reply:
or is it for purposes of blocking? should be enum then - block / focus / same as group?